libs/coin-evm/src/__tests__/unit/api/gasTracker/ledger.unit.test.ts

```diff
@@ -53,100 +55,223 @@ describe("EVM Family", () => {
     });
 
     describe("ledger", () => {
-      it("should return EIP-1559 gas options", async () => {
-        const gasOptions: GasOptions = await getGasOptions({
-          currency: fakeCurrency as CryptoCurrency,
-          options: {
-            useEIP1559: true,
-          },
+      describe("EIP-1559 gas options", () => {
+        it("should return EIP-1559 gas options", async () => {
+          const gasOptions: GasOptions = await getGasOptions({
+            currency: fakeCurrency as CryptoCurrency,
+            options: {
+              useEIP1559: true,
+            },
+          });
+
+          const expectedGasOptions: GasOptions = {
+            slow: {
+              maxFeePerGas: new BigNumber(9),
+              maxPriorityFeePerGas: new BigNumber(1),
+              gasPrice: null,
+              nextBaseFee: new BigNumber(4),
+            },
+            medium: {
+              maxFeePerGas: new BigNumber(10),
+              maxPriorityFeePerGas: new BigNumber(2),
+              gasPrice: null,
+              nextBaseFee: new BigNumber(4),
+            },
+            fast: {
+              maxFeePerGas: new BigNumber(11),
+              maxPriorityFeePerGas: new BigNumber(3),
+              gasPrice: null,
+              nextBaseFee: new BigNumber(4),
+            },
+          };
+
+          expect(gasOptions).toEqual(expectedGasOptions);
         });
 
-        const expectedGasOptions: GasOptions = {
-          slow: {
-            maxFeePerGas: new BigNumber(9),
-            maxPriorityFeePerGas: new BigNumber(1),
-            gasPrice: null,
-            nextBaseFee: new BigNumber(4),
-          },
-          medium: {
-            maxFeePerGas: new BigNumber(10),
-            maxPriorityFeePerGas: new BigNumber(2),
-            gasPrice: null,
-            nextBaseFee: new BigNumber(4),
-          },
-          fast: {
-            maxFeePerGas: new BigNumber(11),
-            maxPriorityFeePerGas: new BigNumber(3),
-            gasPrice: null,
-            nextBaseFee: new BigNumber(4),
-          },
-        };
+        it("should return integer values when EIP1559_BASE_FEE_MULTIPLIER is a float", async () => {
+          setEnv("EIP1559_BASE_FEE_MULTIPLIER", 1.5);
 
-        expect(gasOptions).toEqual(expectedGasOptions);
-      });
+          // @ts-expect-error method is mocked
```

Comment:
Minor but I think you could use `const mockedNetwork = jest.mocked(network) to have right typings`

It's already used somewhere else and will fix all those ts expect error